airbyte-container-orchestrator/src/test/java/io/airbyte/container_orchestrator/config/ContainerOrchestratorFactoryTest.java

```diff
@@ -33,6 +38,10 @@ class ContainerOrchestratorFactoryTest {
   @Inject
   FeatureFlags featureFlags;
 
+  @Bean
+  @Replaces(FeatureFlagClient.class)
```

Comment:
What is the 'Replaces' annotation?

Reply:
This tells Micronaut to use _this_ object to replace any `FeatureFlagClient` object it knows about for dependency injection purposes.  Specifically here I'm telling Micronaut to use the `TestClient` implementation of the `FeatureFlagClient` instead of the `LaunchDarklyClient` or `ConfigFileClient` that it would have provided.

Reply:
Thanks for the explanation.